### ISSUE TITLE
Uncaught TypeError: Cannot read property 'indexOf' of null

### ISSUE THREAD

**Issue by RamanvProjects** [atom/tabs#151]
1. Use split pane
2. Open tabs in second pane
3. Close out of tabs

**Atom Version**: 0.204.0
**System**: Mac OS X 10.10.3
**Thrown From**: [tabs](https://github.com/atom/tabs) package, v0.69.0

### Stack Trace

Uncaught TypeError: Cannot read property 'indexOf' of null

```
At /Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:84

TypeError: Cannot read property 'indexOf' of null
  at Pane.module.exports.Subscriber.unsubscribe (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/emissary/lib/subscriber.js:84:38)
  at Pane.module.exports.Pane.removeItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:363:14)
  at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app.asar/src/pane.js:442:16)
  at HTMLDivElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:183:22)
  at space-pen-ul.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at space-pen-ul.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -6:37.7.0 settings-view:check-for-package-updates (atom-workspace.workspace.scrollbars-visible-when-scrolling.theme-one-light-syntax.theme-one-light-ui)
     -6:04.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -6:03.7.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:45.4.0 run:terminate (ol.tree-view.full-menu.list-tree.has-collapsable-children.focusable-panel)
     -0:38.5.0 run:script (atom-text-editor.editor.vim-mode.insert-mode.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ],
    "disabledPackages": [
      "autocomplete-plus-jedi",
      "atom-beautify",
      "autocomplete-haskell",
      "haskell-ghc-mod",
      "ide-haskell",
      "atom-terminal-panel",
      "language-newlisp",
      "build"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-python, v0.4.1
language-lisp, v0.1.2
language-lua, v0.9.3
language-racket, v0.4.0
language-rust, v0.4.3
language-scala, v1.1.0
language-scala-plus, v1.2.1
love-launcher, v0.5.0
minimap, v4.9.0
processing-language, v0.1.0
script-runner, v1.8.3
terminal-status, v1.6.7
vim-mode, v0.49.0

# Dev
No dev packages
```

**Comment by izuzak** [comment]
@RamanvProjects I'm having trouble reproducing this. Can you reproduce this in [safe mode](https://atom.io/docs/latest/hacking-atom-debugging#check-if-the-problem-shows-up-in-safe-mode)? If so, can you record a GIF which demonstrates what you're doing -- I might be missing something.

**Comment by kevinsawicki** [comment]
This was fixed in Atom 1.0 now that emissary is no longer used for tracking subscriptions in the `Pane` model.
